src/cf-icons/usage.md

```diff
@@ -37,11 +37,11 @@ Overwrite them in your own project by duplicating the variable `@key: value`.
 
 ### Color variables
 
-`$color-` variables are from 18F's
-[U.S. Web Design Standards](https://github.com/18F/web-design-standards/blob/staging/src/stylesheets/core/_variables.scss)
+Color variables referenced in comments are from [cf-core cf-brand-colors.less](https://github.com/cfpb/capital-framework/blob/master/src/cf-core/src/cf-brand-colors.less).
```

Comment:
Is this a URL that _will_ be present? It's 404 for me currently.

Reply:
yup, once the cf-core component is released, that will be the link to the cf-core vars on the master branch.